spec/views/content_assets/edit.html.erb_spec.rb

```diff
@@ -2,16 +2,13 @@
 
 RSpec.describe "content_assets/edit", type: :view do
```

Comment:
I've spoken to @guy-roberts about replacing view specs with cucumber interaction tests down the line. These will need moving over when @dmallett1 has it all setup 😄